cmd/kubeadm/app/cmd/upgrade/node.go

```diff
@@ -108,6 +108,7 @@ func newNodeOptions() *nodeOptions {
 	return &nodeOptions{
 		kubeConfigPath: constants.GetKubeletKubeConfigPath(),
 		dryRun:         false,
+		etcdUpgrade:    true,
```

Comment:
although, the mitigation is to simply pass the CLI flag, not upgrading etcd is not a good idea.
the version difference between 1.15 and 1.16 is "3.3.10 -> 3.3.15", at the time of writing this message.